pkgs/tools/misc/ppfetch/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenvNoCC, lib, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "ppfetch";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "yael333";
+    repo = "ppfetch";
+    rev = "acf00175113db7e4bcd04f40596494360d232fdf";
```

Comment:
It seems the upstream repo does not have tags or releases. As you seem to be the original author and maintainer of the project could you create GitHub releases or tags? This would make managing this Nix package easier as it provides more version transparency.